testing-junit-common/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 apply from: '../gradle/publication.gradle'
```

Comment:
```suggestion

```

This needs to be removed. Common module must not be published

Reply:
I believe it does need to be published, otherwise the artifacts won't be available. We don't produce a 'fat' binary that includes artifacts from `implementation` dependencies

Reply:
Also, if someone wants to use a testing framework other than junit (spek for example) they can use the common module as a good starting point